buildstockbatch/hpc.py

```diff
@@ -111,7 +111,11 @@ def results_dir(self):
     def clear_and_copy_dir(src, dst):
         if os.path.exists(dst):
             shutil.rmtree(dst, ignore_errors=True)
-        shutil.copytree(src, dst)
+        while True:
```

Comment:
Maybe this should have a limit of the number of times it can run and error out so it doesn't hang forever if the file doesn't ever delete and it's not just a weird race condition. I'll fix that.